lib/checker.rb

```diff
@@ -0,0 +1,36 @@
+class Checker
+
+	def check_items(order,menu)
+		order.keys.each do |order_item|
+			unless menu.keys.include?(order_item)
+				raise "Sorry, #{order_item} is not on the menu. Please re-order. (View the menu using .view_menu)"
+			end
+		end
+	end
+
+	def check_price(total,order,menu)
+		price = calculate_price(order,menu)
+		if total > price
```

Comment:
avoid if / else conditionals - use guard clauses, or extract functionality away. In this case your method has too many responsibilities.

avoid presentation strings inside your error messages